metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/bolt/ParserBolt.java

```diff
@@ -117,7 +117,7 @@ public void execute(Tuple tuple) {
       int numWritten = 0;
       if(sensorParserConfig != null) {
         List<FieldValidator> fieldValidations = getConfigurations().getFieldValidations();
-        Optional<List<JSONObject>> messages = parser.parseOptional(originalMessage);
+        Optional<List<JSONObject>> messages = parser.parseOptional(originalMessage, sensorParserConfig);
```

Comment:
Couldn't we have a Configuration Management object - + interfaces that was the 'source of truth' for the configuration?  A default implementation that handled the ZK/Curator changes hand handed out the most recent configuration when asked could then be created.  A different implementation could be from file or from ????? etc.

We could have the access to that object be part of the interface that the parser could use, instead of implicitly passing it around and cluttering the api